api/grpc/grpc.go

```diff
@@ -49,6 +49,8 @@ type Service struct {
 	TraceURL  string    `json:"traceurl"`
 	Handler   adapter.Handler
 	Channel   chan interface{}
+
+	meshes.UnimplementedMeshServiceServer
```

Comment:
A little deviating from the design, ideally this should be registered as a photo method in `Start` function.

Reply:
@kumarabd I think embedding it here is required by grpc now (https://github.com/grpc/grpc-go/issues/3794). I am not sure how to avoid embedding this auto-generated struct into the Service struct. :sweat_smile: